### PR TITLE
PLAT-3879 Add package skeleton

### PR DESCRIPTION
**Pull request by ian-axelrod**
Add the basic files for musekafka package.

PLAT-3879

### CODE REVIEW

**Review by Shikkic — APPROVED** [review]
LGTM

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+from setuptools import find_packages, setup
+
+setup(
+    name="musekafka",
+    use_scm_version={"write_to": "src/musekafka/_version.py"},
+    setup_requires=["setuptools_scm"],
+    description="Python client to simplify work with Kafka consumers and producers.",
+    long_description=open("README.md").read(),
+    long_description_content_type="text/markdown",
+    license="MIT",
```
Comment (Shikkic) [inline setup.py:12]: I thought we used `BSD 3`?

Comment (ian-axelrod) [reply setup.py:12]: Not that I'm aware of. Who stated that?

Comment (Shikkic) [reply setup.py:12]: here is a DM I had with Yusuf awhile back

```
(Also totally unrelated but if I created a Dailymuse OSS lib, what sort of license would I need?)

license is BSD 3-clause: https://github.com/dailymuse/oz/blob/develop/LICENSE
GitHubGitHub
dailymuse/oz
oz - Middleware and scaffolding for tornado-based webapps
5:31
just update the copyright when you import that :slightly_smiling_face:

dcad:spiral_calendar_pad:  5:31 PM
Awesome! Thanks!
```

Oz uses the BS3-3 as well. idk if this is just an old policy but thought I'd mention.

Comment (Shikkic) [reply setup.py:12]: The main difference between the licenses being that

```
but with a 3rd clause that prohibits others from using the name of the project or its contributors to promote derived products without written consent.
```

Which I think we would want that kind of protection right?

**Pull request merged**